src/Components/Layout.tsx

```diff
@@ -0,0 +1,14 @@
+import { Outlet } from 'react-router-dom';
+import { Navigation } from './Navigation';
+
+export const Layout = () => (
+  <>
+    <Navigation />
+
+    <div className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </div>
+  </>
```

Comment:
Would be better to use this container as a content wrapper in pages.

```suggestion
    <Navigation />

    <div className="section">
        <Outlet />
    </div>
  </>
```